### PR TITLE
docs: libraries-bom 13.3.0

### PR DESCRIPTION
**Pull request by suztomo**
@elharo @chingor13 Can you review this?

### CODE REVIEW

**Review by elharo — CHANGES_REQUESTED** [review]
13.3.0

**Review by lesv — APPROVED** [review]

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ There are different strategies to resolve conflicts, but you must understand the
   least hackish approach but it is a lot of work that can require multiple releases
    of multiple libraries to sync everything up. 
 - If you can't modify and push new versions of your dependencies, import
-  `com.google.cloud:libraries-bom:12.1.0` (or a more recent version) and use that to
+  `com.google.cloud:libraries-bom:12.3.0` (or a more recent version) and use that to
```
Comment (elharo) [inline TROUBLESHOOTING.md:102]: 13.3.0

**Review by elharo — CHANGES_REQUESTED** [review]

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (suztomo) [reply TROUBLESHOOTING.md]: Fixed

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ There are different strategies to resolve conflicts, but you must understand the
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>12.1.0</version>
+        <version>12.3.0</version>
```
Comment (suztomo) [inline TROUBLESHOOTING.md]: Fixed.

**Review by suztomo — COMMENTED** [review]
@elharo PTAL.

**Review by elharo — APPROVED** [review]

**Pull request merged**